NorthstarDLL/plugins/plugins.cpp

```diff
@@ -188,8 +188,36 @@ std::optional<Plugin> PluginManager::LoadPlugin(fs::path path, PluginInitFuncs*
 	return plugin;
 }
 
+inline void findPlugins(fs::path pluginPath, std::vector<fs::path>& paths)
```

Comment:
```suggestion
inline void FindPlugins(fs::path pluginPath, std::vector<fs::path>& paths)
```

Reply:
the name was explicitly chosen to match `freeLibrary` in the same file.

Reply:
We use `PascalCase` in the entire codebase, `freeLibrary` should also follow that

Reply:
freeLibrary is a wrapper around FreeLibrary that should probably be generalized and renamed.

Reply:
ik, but thats out of scope for this PR